packages/api/tests/interfaces/http/associations.spec.ts

```diff
@@ -109,4 +123,42 @@ describe("/association", () => {
             expect(response.body).toMatchSnapshot();
         });
     });
+
+    describe("/{identifier}/grants", () => {
+        it("should return grants with siren", async () => {
+            // SIREN must be from an association
+            const SIREN = siretToSiren(OsirisRequestEntityFixture.legalInformations.siret);
+            await rnaSirenPort.insert({ siren: SIREN, rna: OsirisRequestEntityFixture.legalInformations.rna as Rna });
+
+            const response = await request(g.app)
+                .get(`/association/${SIREN}/grants`)
+                .set("x-access-token", await createAndGetAdminToken())
+                .set("Accept", "application/json");
+            expect(response.statusCode).toBe(200);
+
+            const expectAnyRawGrantId = (rawGrant: RawGrant) => ({
```

Comment:
```suggestion
            const expectAnyRawGrantAnyId = (rawGrant: RawGrant) => ({
```
j'ai l'impression que c'est + juste

Reply:
nan mais en fait ça me fait remonter un autre truc; il faut pas renvoyer d'id mongo c'est inutile; mais bon c'est pas le sujet de cette PR. Mais sur le nommage je trouve mon nommage correcte : on parle de remplacer le rawGrant._id` par `expect.any`. C'est le nom du mapper, qui ne prend en paramètre qu'un seul rawGrant et pas un tableau

Reply:
Moi je lisais "id de n'importe quelle raw grant" et pas "raw grant avec n'importe quel id" d'où la précision que je proposais. Mais je peux m'habituer à ta formulation

Reply:
non je pense que tu lis "mal" parce que tu t'attends au résultat alors que moi je décris le mapper (que fait le mapper ? il renvoi UN raw grant avec un expect.any à la place de `_id`, mais est appelé X fois durant le `.map`)